src/models/feed.rs

```diff
@@ -34,6 +35,7 @@ impl TryFrom<Row> for Feed {
                 .map(|s| s.to_owned()),
             last_updated: DateTime::from_str(row.read::<&str, _>("last_updated"))
                 .map_err(|e: chrono::ParseError| RepositoryError::Deserialization(e.into()))?,
+            unread_count: row.read::<Option<i64>, _>("unread_count").unwrap_or_else(|| 0),
```

Comment:
```suggestion
            unread_count: row.read::<u16, _>("unread_count"),
```
I believe we are ok removing the `Option` and reading directly the `u16` value, since the `(SELECT COUNT(*) FROM article a WHERE a.feed_id = f.id AND a.read = 0)` query will return 0 if there are no unread values.

Besides that, if this was a case of using `Option`, given that `0` is a plain value, here is better to use `unwrap_or` and not `unwrap_or_else`. `unwrap_or_else` is usually used when the alternative value has to be computed. The reason behind that is that if the unwrap is never executed, the computation is never executed, so we save some CPU cycles.

Reply:
I've fixed this using clamp in the latest commit because the `sqlite::Value` type cannot be directly converted to a `u16`. I had to read the value as an `i64` and then convert it to a `u16`.